src/fsdocs-tool/BuildCommand.fs

```diff
@@ -2115,3 +2119,13 @@ type WatchCommand() =
 
     [<Option("port", Required = false, Default = 8901, HelpText = "Port to serve content for http://localhost serving.")>]
     member val port = 8901 with get, set
+
+    [<Option("contenthost", Required = false, HelpText = "URI root to inject in static content.")>]
+    member val contenthost = "" with get, set
```

Comment:
Please add documentation for this new setting.
I'm not quite convinced about the flag name and help text.
I also have some slight concerns that invalid input won't be displayed nicely to the end-users.
If you value for this is `blah`, Suave might trip over it.
It is also not quite clear if `http://` or `https://` needs to be included in this.

Reply:
When you say:

> Please add documentation for this new setting.

Do you mean just in the `HelpText`? Or is there somewhere else that I should be adding a more verbose description?

And good point about the invalid input. I added a call to `System.Uri` in the `override` to get its built-in parse errors. Do you think that is helpful enough?

Reply:
The HelpText is a great start and I would like you to also add an entry in https://fsprojects.github.io/FSharp.Formatting/commandline.html (see https://github.com/fsprojects/FSharp.Formatting/blob/main/docs/commandline.md).

Reply:
![image](https://github.com/fsprojects/FSharp.Formatting/assets/2621499/d71aad93-f9ae-41a0-92f8-4c38f8d5c340)

Could you take a look if we can show something friendlier than this?

Reply:
Ah got it! And yes I will look into friendly-ifying the URI error.